sources/packages/web/src/components/students/modals/CancelApplication.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <modal-dialog-base
```

Comment:
For ok/cancel modals we have also the component `confirm-modal` used on 
```
sources\packages\web\src\views\aest\student\StudentAccountApplicationsApproval.vue
```

Reply:
I noticed that this cancel modal is consumed in 3 different places, so no need to change it since it is centralizing the cancel here 😉